ntc_templates/templates/cisco_xr_admin_show_inventory.textfsm

```diff
@@ -1,15 +1,15 @@
 Value Required NAME (.*?)
-Value DESCR (.*)
+Value DESCR (.*?)
 Value PID (\S+)
 Value VID (\S+)
 Value SN (\S*)
 
 Start
   ^NAME:\s+"${NAME}",\s+DESCR:\s+"${DESCR}"\s*$$
   ^\s*Name:\s+${NAME}\s+Descr:\s+${DESCR}\s*$$
-  ^\s*PID:\s+${PID}\s*,\s+VID:\s+${VID}\s*,\s+SN:\s*${SN}$$ -> Record
-  ^\s*PID:\s+${PID}\s+VID:\s+${VID}\s+SN:\s+${SN}$$ -> Record
+  ^\s*PID:\s+${PID}\s*,\s+VID:\s+${VID}\s*,\s+SN:\s*${SN} -> Record
```

Comment:
This either shouldn't be needed or changed to:

`^\s*PID:\s+${PID}\s*,\s+VID:\s+${VID}\s*,\s+SN:\s*${SN}\s*$$`

Reply:
again this should be to there as well for the classic or virtual XR. you can check the script if you like to test it 

Please also refer to my script, I created to test this in the issue #1396

Reply:
What is causing @itdependsnetworks suggestion to not work?

Reply:
> This either shouldn't be needed or changed to:
> 
> `^\s*PID:\s+${PID}\s*,\s+VID:\s+${VID}\s*,\s+SN:\s*${SN}\s*$$`

updated as requested.

Reply:
Please let me know if there is any other changes required ? @itdependsnetworks